src/drm/drm.c

```diff
@@ -197,28 +222,69 @@ int main(int argc, char** argv)
         usage = 1;
     }
 
+    /* print usage if we need to */
+    if (usage) {
+        if (rank == 0) {
+            print_usage();
+        }
+        mfu_finalize();
+        MPI_Finalize();
+        return 1;
+    }
+
     /* create new mfu_file objects */
     mfu_file_t* mfu_file = mfu_file_new();
 
-    /* paths to walk come after the options */
-    int numpaths = 0;
+    char** argpaths = &argv[optind];
+
+    /* The remaining arguments are treated as paths */
```

Comment:
@daltonbohning is this going to work if multiple paths are passed in with different pools?

Reply:
@dsikich Whether they are in the same pool or different pools, mfu_flist_walk_param_paths would print:
"ERROR: Only one source can be specified when using DAOS".
And only the "source" mfu_file is being passed to daos_setup, so daos_setup only looks at the first path.
Ultimately, we'll probably want to be able to support a LIST of daos paths in a general sense for some of the utilities, but for now only a single daos path is supported per mfu_file.

Reply:
@daltonbohning ok, I thought it was setup for only one but was just making sure in case someone passes multiple. I asked at one point early on if we should support multiple paths, and at the time they were not interested and just thought they could run the tool multiple times for each path. It is something we can ask about again as we get into more of the refinements we've been talking about, since it doesn't really hurt to have it as an option.

Reply:
@dsikich I agree it would be nice to have, but we would probably need some amount of refactoring, etc. to get it working fully.